.changeset/flat-boxes-build.md

```
@@ -0,0 +1,5 @@
+---
+'@neshca/cache-handler': minor
+---
+
+Updated the documentation and peer dependencies to explicitly state that only Next.js versions `13.5.x` and `14.x.x` are supported. Modified the default `estimateExpireAge` function to perform the calculation as `(staleAge) => staleAge * 1.5`.
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)